packages/fdb-debugger/src/index.ts

```diff
@@ -43,6 +43,8 @@ export interface ConsoleTrace {
   message: any[];
 }
 
+const FBOS3_EVAL_QUIRK = /^[a-zA-Z]+ \d+\.31\.1\.\d+$/;
```

Comment:
Character classes FTW!
```suggestion
const FBOS3_EVAL_QUIRK = /^\w+ \d+\.31\.1\.\d+$/;
```

Reply:
I was intentionally being more strict, but maybe it's unnecessary...

Reply:
Oops, I thought that `\w` was equivalent. Maybe I'm mixing it up with character classes from another language's regexes.